WcaOnRails/app/views/competitions/_index_table.html.erb

```diff
@@ -0,0 +1,27 @@
+<table class="table competitions-table table-striped table-hover">
+  <tbody>
+    <tr>
+      <th colspan=2><%= title %></th>
+    </tr>
+    <% if competitions.length > 0 %>
+      <% competitions.each_with_index do |competition, index| %>
+        <tr class="<%= competition.is_over? ? "past" : "not-past" %>">
+          <td><span style="white-space: nowrap;"><%= date_range(competition.start_date, competition.end_date, separator: '-', show_year: false) %></span>, <%= competition.year %></td>
+          <td>
+            <p>
+              <%= link_to competition.name, competition_path(competition) %>
+              <br /><span style="font-size: smaller;"><strong><%= competition.country.name %></strong>, <%= competition.cityName %>
```

Comment:
I actually removed the smaller thing, since it looked a little too smal for me. I'll put this up at staging soon so you can take a look.